index.html

```diff
@@ -196,9 +286,30 @@ <h1>About
         <li class='menu'>Skills<span class='arrow rarrow'></span><span class='arrow darrow'>-</span></li>
       </ul>
 
+    </div>
+
+  </section>
+  <section class="contactform">
```

Comment:
- [ ] Kindly let us stick to the Figma design as possible, your contact section has some space from the left and right which is not on the Figma design, see the screenshots below 👇 

- Figma design
![image](https://user-images.githubusercontent.com/55465133/129770614-ed9f0261-10d2-44b8-b333-3e1aa97e78ed.png)

- Your section
![image](https://user-images.githubusercontent.com/55465133/129770528-5f268bcd-e7d5-42fa-8b69-7d050684984a.png)

Can you please fix this issue 👍

Reply:
- [ ] Kindly make sure you did the first reviewer requested changes. Please try to fix the UI of your contact form in order to make it the same as the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1 )one. I think on the mobile screen you should give full width to your contact form section in order to make your design in full width. 
- This is the design you did
![](https://res.cloudinary.com/dflkwvvnz/image/upload/v1629262913/scrnli_8_18_2021_10-00-47_AM_jpmwcl.png)

- As you see there is a white background on the right side and also `contact me` text should be in one line. And also give some padding to your inputs in order to make your UI same as the one in Figma.